prog/vm/host_nexus.rb

```diff
@@ -245,9 +245,26 @@ def get_boot_id
   end
 
   def available?
-    sshable.cmd("true")
-    true
-  rescue
+    devices = storage_devices.flat_map { |sd| sd.unix_devices.split(',') }
```

Comment:
I think you can extract this logic into a function within vmhost and use it in both HostNexus and VmHost

Reply:
done